share/spice/symbolab/subtitle_content.handlebars

```diff
@@ -0,0 +1 @@
+Solution for: <span {{#if problemHasLatex}}class="mathquill-embedded-latex"{{/if}}>{{problem}}</span>
```

Comment:
Once this is wrapped in an `h4` we can remove the CSS that adds a `margin` to the title :+1: